src/dev-spaces/azext_dev_spaces/__init__.py

```diff
@@ -16,8 +16,7 @@ def __init__(self, cli_ctx=None):
         dev_spaces_custom = CliCommandType(
             operations_tmpl='azext_dev_spaces.custom#{}')
         super(DevspacesExtCommandLoader, self).__init__(cli_ctx=cli_ctx,
-                                                        custom_command_type=dev_spaces_custom,
-                                                        min_profile='2017-03-10-profile')
+                                                        custom_command_type=dev_spaces_custom)
```

Comment:
@limingu can you verify this works correctly on your min_profile-less branch?

Reply:
Yes, that is correct.